env/env.go

```diff
@@ -7,11 +7,32 @@ import (
 	"net/url"
 	"os"
 	"strconv"
+	"strings"
 	"time"
 
 	"github.com/pkg/errors"
 )
 
+func IsTrue(s string) bool {
+	s = strings.ToLower(s)
+	if s == "1" || s == "t" || s == "true" {
+		return true
+	}
+	return false
+}
```

Comment:
I don't think we need `IsTrue` to be exported or even split off into a separate function.

Reply:
This was done so I wouldn't need to duplicate this in another package. But if you don't think it should be exported, I can remove that and just duplicate the code in that other package.